Migration.md

```diff
@@ -5,6 +5,13 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Gazebo Physics 8.X to 9.X
+
+### Deprecations
+
+* The macro `GZ_PHYSICS_ENGINE_INSTALL_DIR` is deprecated. Use
+`gz::physics::getEngineInstallDir()` instead.
```

Comment:
should we just remove the macro like other gz packages?

Reply:
How about now?  e1e5bae